concept/thing/attribute.feature

```diff
@@ -173,6 +174,22 @@ Feature: Concept Attribute
     When $x = attribute(birth-date) as(datetime) get: 1990-01-01 11:22:33
     Then attribute(birth-date) get instances contain: $x
 
+  Scenario: Attribute with value type datetime can be inserted in one timezone and retrieved in another timezone with no change in the value
```

Comment:
Shorten: `Datetime attribute can be inserted in one timezone and retrieved in another with no change in the value`